frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -55,6 +56,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       id: 'persistent-storage',
       title: 'OCS PV',
       required: CEPH_FLAG,
+      blocker: INDEPENDENT_FLAG,
```

Comment:
```suggestion
      disallowed: INDEPENDENT_FLAG,
```

Reply:
This flag isn't owned by this plugin, so we could import it from the other plugin as a constant.